torch/onnx/__init__.py

```diff
@@ -55,25 +75,28 @@ def _export(*args, **kwargs):
 
 
 def export(
-    model,
-    args,
-    f,
-    export_params=True,
-    verbose=False,
-    training=TrainingMode.EVAL,
-    input_names=None,
-    output_names=None,
-    operator_export_type=OperatorExportTypes.ONNX,
-    opset_version=None,
-    do_constant_folding=True,
-    dynamic_axes=None,
-    keep_initializers_as_inputs=None,
-    custom_opsets=None,
-    export_modules_as_functions=False,
-):
-    r"""
-    Exports a model into ONNX format. If ``model`` is not a
-    :class:`torch.jit.ScriptModule` nor a :class:`torch.jit.ScriptFunction`, this runs
+    model: Union[torch.nn.Module, torch.jit.ScriptModule, torch.jit.ScriptFunction],
+    args: Union[Tuple[Any, ...], torch.Tensor],
+    f: Union[str, BinaryIO],
+    export_params: bool = True,
+    verbose: bool = False,
+    training: TrainingMode = TrainingMode.EVAL,
+    input_names: Optional[Sequence[str]] = None,
+    output_names: Optional[Sequence[str]] = None,
+    operator_export_type: OperatorExportTypes = OperatorExportTypes.ONNX,
+    opset_version: Optional[int] = None,
+    do_constant_folding: bool = True,
+    dynamic_axes: Optional[
+        Union[Mapping[str, Mapping[int, str]], Mapping[str, Sequence[int]]]
+    ] = None,
+    keep_initializers_as_inputs: Optional[bool] = None,
+    custom_opsets: Optional[Mapping[str, int]] = None,
+    export_modules_as_functions: Union[bool, Collection[torch.nn.Module]] = False,
```

Comment:
`Collection[torch.nn.Module]` -> `Collection[Type[torch.nn.Module]]` would be more accurate.

Reply:
Done